### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by aliciapaz**
### Main features 
- Display prompts for users: player's names
- "Flip a coin" to randomly choose which player goes first
- Display prompts for users: input a number between 1 - 9 to choose a position
- Display game board and updates it after every turn
- Print error message when input is not a number between 1 -9
- Print error message when input is a position already taken

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
**CHANGES REQUIRED**
Hi @aliciapaz @RailonA . This is great work. I especially like the part of selecting a random player👏.
However some changes are required.
- The console should display the name of the player who is going to play.
- The board should be displayed immediately after the game randomly chooses the player.
- Player a can choose the position that player b already chose and the key gets changed. Please fix that.
- The console is displaying all positions, not the positions that are available or empty. Please fix that.

[REQUIRED] game tells which player turn it is.
[REQUIRED] game asks the player to select from available moves
[REQUIRED] game informs player if the selected move is invalid
[OPTIONAL] Try separating logic of game and use instance variable instead of class variables. 
For example: You can create a Player class that holds the information of a player then use it in the TicTacToe class.

Make the required changes and ask for another review.

If you have any question, feel free to contact me on slack. (username: NTWALI Heritier)

**Comment by aliciapaz** [comment]
-  game tells which player turn it is.
- game displays available positions in one board and player's symbols on another board
- game informs player if the selected move is invalid

File: .rubocop.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ Layout/LineLength:
 Metrics/MethodLength:
   Max: 20
 Metrics/AbcSize:
-  Max: 50
+  Max: 60
```
Comment (BrianSammit) [inline .rubocop.yml]: Change this line for the original one (Max: 50)

Screenshot from 2021-01-13 18-54-31

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED :recycle: 
Hello team!! :smile: Great job working on the changes :clap: 
There are only a few changes that you have to work on.
---------------------------------------------------------------------------------------------------------------

- [x] Don't change the rubocop.yml file use the original one. 
Screenshot from 2021-01-13 18-54-31
- [x] Use a loop for the game. when I made 9 moves the game finished which means that the invalid moves count and the game finish before I put all the marks on the board. 
-----------------------------------------------------------------------
Made these changes and request another review.
Feel free to contact me on slack. (username: Brian Sammit Cruz Rodriguez) 
HAPPY CODING :+1: 

MB5W4D

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

### I really like your project! Congrats team, you guys did an excellent job, you have my approval to go ahead and merge this pull request!

⚠ Please update your readme;


## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**